content/docs/start/data-and-model-versioning.md

```diff
@@ -250,16 +246,13 @@ the <abbr>workspace</abbr> efficiently to match them.
 
 In cases where you process very large datasets, you need an efficient mechanism
 (in terms of space and performance) to share a lot of data, including different
-versions of itself. Do you use a network attached storage? Or a large external
-volume?
-
-While these cases are not covered in the Get Started, we recommend reading the
-following sections next to learn more about advanced workflows:
+versions. Do you use network attached storage (NAS)? Or a large external volume?
+You can learn more about advanced workflows using these links:
 
 - A shared [external cache](/doc/use-cases/shared-development-server) can be set
   up to store, version and access a lot of data on a large shared volume
   efficiently.
 - A quite advanced scenario is to track and version data directly on the remote
-  storage (e.g. S3). Check out
-  [Managing External Data](https://dvc.org/doc/user-guide/managing-external-data)
+  storage (e.g. S3). See
+  [managing external data](https://dvc.org/doc/user-guide/managing-external-data)
```

Comment:
"Check out" was OK ("See" is fine too).
Capitalized title was correct though.

Reply:
wouldn't title case unless proper nouns (even in ref. to a doc which uses title case for its, well, title)

Reply:
> We don't title case <abbr>s

That's a website feature (tooltips), not exactly the same as a link to a titled document. But yeah I guess we could capitalize all those tooltips to be extra correct. Let's not tho 😬 

Without capitalization here it looks like we're literally saying "See managing external data," which doesn't look like a sentence to me. "See [Any Doc Title]," seems right?

Reply:
Or if the capitalized link gets you you we can rephrase to something like "See [this doc] on managing external data" but I wouldn't bother, we have lots of capitalized links like this everywhere.

Reply:
"See Proper Noun" is fine but I don't personally like awarding full proper noun status to sections of documentation. In general I'd emphasise (using e.g. italics) but - since the link already provides an underline - didn't think further emphasis was needed.

Happy to revert all links to title case if you prefer though.

It is definitely wrong to title case `<abbr>`s -- they're simply tooltip hints to dictionary definitions.

Reply:
> Happy to revert all links to title case if you prefer

Please, I think we should revert these as long as we use the exact page titles for the links. Thanks

Reply:
? I thought this was already reverted/resolved...